frontend/app/.server/locales/protected-en.json

```diff
@@ -101,11 +101,20 @@
     "required-request": "Type of request is required."
   },
   "current-name": {
-    "page-title": "Current name",
-    "page-description": "The current name fields are pre-filled from the primary identity document. Review and update them to the applicant's current name(s) if needed. These names will be used for their SIN.",
-    "first-name": "Current first name(s)",
-    "middle-name": "Current middle name(s)",
-    "last-name": "Current last name",
+    "page-title": "Name for SIN",
+    "recorded-name": {
+      "description": "The following name is recorded on the applicant's primary identity document:",
+      "first-name": "First name(s): ",
+      "middle-name": "Middle name(s): ",
```

Comment:
Remove the trailing space here and add it to the .tsx file (if it's actually required).